pkg/drivers/kic/oci/network_create.go

```diff
@@ -221,7 +221,6 @@ func CreateNetworkWithIPFamily(ociBin, networkName, subnet, subnetv6, staticIP,
 			klog.Infof("%s network %s %s created", ociBin, networkName, p.CIDR)
 			return gw, nil
 		}
-		// don't retry if error is not address is taken
```

Comment:
There is a second "kic: plumb ip-family and ipv6 for docker bridge" commit that could be merged with the first one.